Content/Items/Pickups/GrimoirePickup.cs

```diff
@@ -79,7 +87,7 @@ public override bool OnPickup(Player player)
 
 	public override void PostDrawInInventory(SpriteBatch spriteBatch, Vector2 position, Rectangle frame, Color drawColor, Color itemColor, Vector2 origin, float scale)
 	{
-		if (Affixes.Count > 0)
+		if (this.GetInstanceData().Affixes.Count > 0)
```

Comment:
"this" doesn't need to be here, dose it?

Reply:
Yes it does; it's an extension method.